.github/workflows/deployment.yml

```diff
@@ -26,3 +26,28 @@ jobs:
         with:
           user: __token__
           password: ${{ secrets.PYPI_API_TOKEN }}
+
+      - name: Login to Docker Hub
+        id: login-dockerhub
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
```

Comment:
We should use a more maintainable tag, in this case `v2` (the current latest version is `v2.1.0`, so we should limit to that major version)

Source: https://github.com/marketplace/actions/docker-login#docker-hub

Reply:
I have implemented this